bokeh/embed/standalone.py

```diff
@@ -88,18 +87,12 @@ def autoload_static(model, resources, script_path):
     with OutputDocumentFor(models):
         (docs_json, [render_item]) = standalone_docs_json_and_render_items([model])
 
-    bundle = bundle_all_models() or ""
-    script = script_for_render_items(docs_json, [render_item])
+    bundle = bundle_for_objs_and_resources(None, resources)
+    bundle.add(Script(script_for_render_items(docs_json, [render_item])))
 
-    (modelid, elementid) = list(render_item.roots.to_json().items())[0]
+    (_, elementid) = list(render_item.roots.to_json().items())[0]
 
-    js = wrap_in_onload(AUTOLOAD_JS.render(
-        js_urls = resources.js_files,
-        css_urls = resources.css_files,
-        js_raw = resources.js_raw + [bundle, script],
-        css_raw = resources.css_raw_str,
-        elementid = elementid,
-    ))
+    js = wrap_in_onload(AUTOLOAD_JS.render(bundle=bundle, elementid=elementid))
```

Comment:
So, I hate the name "autoload_whatever". Instead of complicating the existing template to handle this new format, can we just make a new thing with a new name and deprecate the autoload stuff?

Reply:
This is what I wanted to avoid and make minimal, backwards compatible changes, and leave deeper changes for 2.0 or later. At this point I intent to redesign embed and resources from scratch at some point.